website/content/docs/discovery/dns.mdx

```diff
@@ -444,33 +473,46 @@ If you need more complex behavior, please use the
 
 ### Service Virtual IP Lookups
 
-To find the unique virtual IP allocated for a service:
+To find the unique virtual IP allocated for a [Connect-capable](/docs/connect) service:
```

Comment:
Note to self before resubmitting PR: add cross-references to when to use virtual IP lookups back in